docs/product-assembly_installing-dm-on-premise/src/main/asciidoc/main.adoc

```diff
@@ -37,36 +30,30 @@ include::product-installation-guide/ba-dm-supported-component-versions-ref.adoc[
 * Why should you do it? 
 * How do you implement it?
 
-include::product-installation-guide/clustering-con.adoc[leveloffset=+3]
-include::product-installation-guide/clustering-git-repo-con.adoc[leveloffset=+3]
-include::product-installation-guide/clustering-maven-repo-con.adoc[leveloffset=+3]
+//include::product-installation-guide/clustering-con.adoc[leveloffset=+3]
+//include::product-installation-guide/clustering-git-repo-con.adoc[leveloffset=+3]
+//include::product-installation-guide/clustering-maven-repo-con.adoc[leveloffset=+3]
 include::product-installation-guide/roles-users-con.adoc[leveloffset=+2]
 
-COMMENT: Still only admin and analyst roles?
-
-=== Security (BRMS 6.4 IG Section 6.2 ?)
-COMMENT: Do we still need this section, Enabling the Security Manager?
-https://access.redhat.com/documentation/en-us/red_hat_jboss_brms/6.4/html-single/installation_guide/#security_manager
-
-== Installing {PRODUCT}
 COMMENT: We need to make sure that this section sets up a real production environment, including high availability and clustering. Some information about configuration topics needed from previous chapter 3 and the appendix in the BRMS admin/cfg guide.
 
-include::product-installation-guide/installing-on-eap-con.adoc[leveloffset=+2]
+include::product-installation-guide/installing-on-eap-con.adoc[leveloffset=+1]
 include::product-installation-guide/install-download-proc.adoc[leveloffset=+2]
 include::product-installation-guide/assembly_installing-on-eap-installer.adoc[leveloffset=+2]
 include::product-installation-guide/assembly_installing-on-eap-deployable.adoc[leveloffset=+2]
+include::product-installation-guide/eap-ba-dm-run-proc.adoc[leveloffset=+2]
 
 == Configuring Clustering
 
 === Clustering on Red Hat JBoss EAP 
-To install {PRODUCT} in the clustered mode, the JAR installer provides a sample setup. You can configure clustering with the deployable ZIP for EAP as well.
 
-include::product-installation-guide/clustering-jar-installer-proc.adoc[leveloffset=+3]
-include::product-installation-guide/cluster-custom-config-proc.adoc[leveloffset=+3]
-include::product-installation-guide/cluster-starting-proc.adoc[leveloffset=+3]
+COMMENT: to be written
+COMMENT: Do we need to tell users how to cluster on other containers?
```

Comment:
we need to tell them we support on-premise clustering and openshift clustering with NFS.

There is NO zookeeper NO helix in 7.0.